src/algorithms/calorimetry/CalorimeterClusterRecoCoGConfig.h

```diff
@@ -14,6 +14,13 @@ namespace eicrecon {
         double sampFrac = 1.;
         double logWeightBase = 3.6;
 
+        //optional:  have the log weight base depend on the energy
+        // logWeightBase+logWeightBase_lin*l+logWeightBase_quad*l*l
+        // where l = log(cl.getEnergy()/logWeightBase_Eref)
+        double logWeightBase_lin=0;
+        double logWeightBase_quad=0;
+        double logWeightBase_Eref=50;
+
```

Comment:
How about `std::vector<double> logWeightCoeffs{3.6}` with index = power of `l`? Fewer variables and combines the info that logically goes together.